testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -54,8 +215,9 @@ public void testTrinoReadingTableCreatedByNativeSpark()
                         "USING ORC\n" +
                         "CLUSTERED BY (a_string)\n" +
                         "INTO 4 BUCKETS\n" +
-                        // Hive requires "original" files of transactional tables to conform to the bucketed tables naming pattern
```

Comment:
Maybe we should also add some decimals and timestamps to the bucketed table scenario, just to be sure it works?

Reply:
i think the point of bucketed table test is to verify the table doesn't show as bucketed in trino.
i don't think additional data columns, or file formats coverage, change much, that's why i decided to have these tests separate.